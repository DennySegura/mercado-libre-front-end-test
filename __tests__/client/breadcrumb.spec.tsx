import 'jest';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import BreadCrumb from '../../src/app/components/breadcrumb';

describe('Test suites for react components', () => {
  it('It should render <BreadCrumb />', () => {
    const categories = ['Electr√≥nica, Audio y Video','Televisores','TV 4K','50"','Samsung'];
    const _breadcrumb = renderer.create(<BreadCrumb categories={categories}/>);
    let breadcrumb_tree = _breadcrumb.toJSON();
    expect(breadcrumb_tree).toMatchSnapshot();
  });
});
