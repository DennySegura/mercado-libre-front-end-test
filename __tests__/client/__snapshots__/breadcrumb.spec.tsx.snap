// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suites for react components It should render <BreadCrumb /> 1`] = `
<ul
  className="breadcrumb--container pv"
>
  <li>
    Electr√≥nica, Audio y Video
  </li>
  <li>
    Televisores
  </li>
  <li>
    TV 4K
  </li>
  <li>
    50"
  </li>
  <li>
    Samsung
  </li>
</ul>
`;
